web-api/compose.single-machine-deploy.yml

```diff
@@ -45,47 +41,50 @@ services:
       retries: 5
 
   backoffice:
-    image: xaynetci/xayn_discovery_ingestion_service:main-230831084031-f320e8c-multilingual_e5_small_v0001-amd64
+    image: xaynetci/xayn_discovery_ingestion_service:main-231019104503-d06d92c-e5_small_v2_v0001-amd64
     mem_limit: 1.5g
     cpus: 2
     restart: always
     environment:
       XAYN_WEB_API__STORAGE__POSTGRES__BASE_URL:  "postgres://${POSTGRES_USER:-user}:${POSTGRES_PASSWORD:-pw}@postgres:5432/xayn"
       XAYN_WEB_API__STORAGE__ELASTIC__URL: "http://elasticsearch:9200/"
-      XAYN_WEB_API__NET__BIND_TO: "0.0.0.0:8000"
+      XAYN_WEB_API__NET__BIND_TO: "0.0.0.0:8081"
```

Comment:
We could avoid to change the port and expose it to another mapping but I think it's easier to read and less to misuse if they are the same

Reply:
The docker network internal port is somewhat irrelevant so we can use whatever is the most convenient.